pytorch_lightning/core/datamodule.py

```diff
@@ -67,28 +66,28 @@ def wrapped_fn(*args, **kwargs):
         obj = args[0]
 
         # If calling setup, we check the stage and assign stage-specific bool args
-        if fn.__name__ == 'setup':
+        if fn.__name__ == "setup":
 
             # Get stage either by grabbing from args or checking kwargs.
             # If not provided, set call status of 'fit' and 'test' to True.
             # We do this so __attach_datamodule in trainer.py doesn't mistakenly call setup('test') on trainer.test()
-            stage = args[1] if len(args) > 1 else kwargs.get('stage', None)
+            stage = args[1] if len(args) > 1 else kwargs.get("stage", None)
 
-            if stage == 'fit' or stage is None:
+            if stage == "fit" or stage is None:
                 obj._has_setup_fit = True
 
-            if stage == 'test' or stage is None:
+            if stage == "test" or stage is None:
                 obj._has_setup_test = True
 
-        if fn.__name__ == 'prepare_data':
+        if fn.__name__ == "prepare_data":
             obj._has_prepared_data = True
 
         return fn(*args, **kwargs)
 
     return wrapped_fn
 
 
-class LightningDataModule(DataHooks, metaclass=_DataModuleWrapper):
+class LightningDataModule(DataHooks, CheckpointHooks, metaclass=_DataModuleWrapper):
```

Comment:
this seems the only change in this file, pls rever all the others...

Reply:
I’m happy to split out the formatting changes to a separate

Reply:
@Borda I've split the formatting changes into separate PRs to make this easier to review:
#3574 
#3575 
#3576 
#3579

Reply:
we have not agreed on using Black as Black is no standard, just another package wit many issues and even violating pep8, also this formatting which is not really needed crashes plenty already open PRs... 
so pls don't do this formating especially ' -> "

Reply:
I see it is merged it will make bad da for many people... :(